mail_optional_follower_notification/models/mail_thread.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    def _notify_compute_recipients(self, message, msg_vals):
+        """ Compute recipients to notify based on subtype and followers. This
+        method returns data structured as expected for ``_notify_recipients``. """
+        recipient_data = super()._notify_compute_recipients(message, msg_vals)
+        if not self.env.context.get("notify_followers", False):
```

Comment:
```suggestion
        if 'notify_followers' in self.env.context and \
                not self.env.context.get("notify_followers", False):
```

Reply:
When I send a direct message via chat, notify_followers is not in the context, and messages are never sent.

Adding this condition fixes the issue.